app-web-frontend/src/Login/HealthProviderLogin.js

```diff
@@ -0,0 +1,185 @@
+import React, { useState } from 'react';
+import './Login.css';
+import {useNavigate} from "react-router-dom";
+import axios from "axios";
+
+function Footer() {
+    return (
+        <footer className="App-footer">
+            <p>© 2024 Emotional Support</p>
+        </footer>
+    );
+}
+
+function HealthProviderLogin() {
```

Comment:
Something like small underlined "Back" link in the bottom would be great to return to LoginChoice screen.